eve/main.yml

```
@@ -246,6 +248,8 @@ stages:
             ssh -F bootstrap_ssh_config bootstrap
             sudo mkdir -p /etc/metalk8s &&
             scp -F bootstrap_ssh_config bootstrap.yaml bootstrap: &&
+            ssh -F bootstrap_ssh_config
+                'sh -c "sed -i s/@HOSTNAME@/$(hostname)/ boostrap.yaml"' &&
```

The error probably comes from the typo:

```suggestion
                'sh -c "sed -i s/@HOSTNAME@/$(hostname)/ bootstrap.yaml"' &&
```